geospaas/base_viewer/templates/base_viewer/template_for_base.html

```diff
@@ -54,9 +54,11 @@
 <form method="post" width='100%' height='100%'>
     {% csrf_token %}
 
-    {{SpatialSearchForm.as_p}}
 
-    {{TimeAndSourceForm.as_p}}
+
+    {{OverallForm.as_p}}
```

Comment:
It is better not to hardcode here a name of the form class. Because when we override IndexView we will have to change the name of the form class. Then we would also have to change the template. But if in opposite we keep a generic name (say 'form') here (and, hence, in set_context) then we are able to reuse this template.
```suggestion
    {{form.as_p}}
```